src/accelerate/data_loader.py

```diff
@@ -326,12 +326,21 @@ class DataLoaderDispatcher(DataLoader):
     """
 
     def __init__(self, dataset, split_batches: bool = False, **kwargs):
+        shuffle = False
+        if version.parse(torch.__version__) >= version.parse("1.11.0"):
```

Comment:
```suggestion
        if version.parse(torch.__version__) >= version.parse("1.11.99"):
```
Just in case we get many micro releases ;-)

Reply:
Arf, review before coffee. I read > and thought for a minute it needed PyTorch 1.12, you can ignore :-)